OracleWebLogic/samples/12212-domain/container-scripts/domain.properties

```diff
@@ -0,0 +1,10 @@
+DOMAIN_NAME=abc_domain
+ADMIN_PORT=7001
+ADMIN_HOST=wlsadmin
+CLUSTER_NAME=DockerCluster
+PRODUCTION_MODE=dev
+ADMIN_PASSWORD=welcome4
```

Comment:
Yes this would be the perfect spot to generate the password but this is not a shell script but a env file.  How do I call  a function that auto-generates the password at runtime for the AdminServer and then when the Managed Server runs it uses the same password generated for the Admin Server.